### PR TITLE
Added comment form to the bottom of comments list.

### PR DESCRIPTION
**Pull request by joaojusto**
Added comment form to the bottom of comments list instead of showing the sliding screen, when on desktop
screen shot 2015-01-23 at 11 12 57
screen shot 2015-01-23 at 11 13 07

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -33,13 +33,6 @@
     t.integer  "comments_count", default: 0,    null: false
   end
 
-  create_table "notifications", force: :cascade do |t|
```
Comment (gabrielpoca) [inline db/schema.rb]: Why are you removing this lines?

File: app/assets/stylesheets/components/_sliding_screen.css.sass [hunk 2]
```diff
@@ -1,5 +1,4 @@
 .SlidingScreen
-  left: 0
```
Comment (gabrielpoca) [inline app/assets/stylesheets/components/_sliding_screen.css.sass]: This must be necessary, the sliding screen take the full page

Comment (joaojusto) [inline app/assets/stylesheets/components/_sliding_screen.css.sass]: Nope, the left:0 isn't doing nothing.

File: app/assets/javascripts/app/components/comment_form/comment_form.js [hunk 3]
```diff
@@ -13,7 +13,8 @@
       scope: {
         toggle: "&toggle",
         comment: "=comment",
-        discussion: "=discussion"
+        discussion: "=discussion",
+        isMobile: "=mobile"
```
Comment (gabrielpoca) [inline app/assets/javascripts/app/components/comment_form/comment_form.js]: Why is this component receiving this property? Why can't you put `Device` on the dependencies?

File: db/schema.rb (hunk 1 above)
Comment (joaojusto) [inline db/schema.rb]: This was during the rebase, let me check if this was removed from master

File: app/assets/javascripts/app/discussions/show.html.slim [hunk 4]
```diff
@@ -35,7 +35,8 @@
               a ng-click="ctrl.toogleDiscussionState(ctrl.discussion)" reopen discussion
 
       md-content.md-padding md-theme="white"
-        div comments-list="ctrl.discussion.comments" comment-count="ctrl.discussion.comments_count" toggle="ctrl.openEditForm(comment)"
+        div layout="column" comments-list="ctrl.discussion.comments" comment-count="ctrl.discussion.comments_count" toggle="ctrl.openEditForm(comment)"
+        comment-form ng-hide="ctrl.device.isMobile()" toggle="ctrl.toggleCommentForm()" mobile="ctrl.device.isMobile()" comment="ctrl.comment" discussion="ctrl.discussion"
 
-sliding-screen show="ctrl.showNewComment" hide-bottom=""
-  comment-form toggle="ctrl.toggleCommentForm()" discussion="ctrl.discussion" comment="ctrl.comment"
+sliding-screen show="ctrl.device.isMobile()" hide-bottom=""
```
Comment (gabrielpoca) [inline app/assets/javascripts/app/discussions/show.html.slim]: Why are you controlling the show both here and on the comment_form?

File: app/assets/stylesheets/components/_sliding_screen.css.sass (hunk 2 above)
Comment (gabrielpoca) [inline app/assets/stylesheets/components/_sliding_screen.css.sass]: The default value auto. [Here](http://www.w3schools.com/cssref/pr_pos_left.asp).

Comment (joaojusto) [inline app/assets/stylesheets/components/_sliding_screen.css.sass]: Ok

File: app/assets/javascripts/app/components/comment_form/comment_form.js (hunk 3 above)
Comment (joaojusto) [inline app/assets/javascripts/app/components/comment_form/comment_form.js]: You are right, I did this before creating the service

File: app/assets/javascripts/app/discussions/show.html.slim (hunk 4 above)
Comment (joaojusto) [inline app/assets/javascripts/app/discussions/show.html.slim]: Yes, but now that you are pointing this I think I will create another template or another directive for the desktop form

**Comment by joaojusto** [comment]
@gabrielpoca bump

File: app/assets/javascripts/app/discussions/discussion.js [hunk 5]
```diff
@@ -5,10 +5,12 @@
     .module('council.discussions')
     .controller('DiscussionCtrl', DiscussionCtrl);
 
-  function DiscussionCtrl(discussion) {
+  function DiscussionCtrl(Device, $location, $anchorScroll, discussion) {
     var ctrl = this;
 
+    ctrl.isMobile = Device.isMobile();
     ctrl.pageReady = false;
+    ctrl.scrollTo = scrollTo;
```
Comment (gabrielpoca) [inline app/assets/javascripts/app/discussions/discussion.js]: Is this needed?

Comment (joaojusto) [inline app/assets/javascripts/app/discussions/discussion.js]: No, I forget this here

**Comment by gabrielpoca** [comment]
:+1:

**Pull request merged**